projects/fsastorefrontlib/src/recipes/storefront.module.ts

```diff
@@ -88,7 +88,7 @@ import { defaultFSGlobalMessageConfigFactory } from '../core/global-message-conf
     }),
     ConfigModule.withConfig({
       date: {
-        format: 'yyyy-mm-dd',
+        format: 'dd/MM/yyyy',
```

Comment:
You can create default-date.config.ts and then provide it instead of this code block.  As this is a really small configuration, so it's up to you if you want to do it. 
From structure perspective, it is always better to have custom configurations in separate files. Also, customers will have a better overview of things that need to be overriden.

Reply:
I'll create config file